### DISCUSSION TITLE
no response for mouse or keyboard click on macbookpro m2max

### DISCUSSION THREAD

**Discussion by yswtrue** [Genymobile/feedback#86] [Genymotion Desktop] [answered]
I am using MacOS 14.3.1 (23D60)
and genymotion 3.6.0
when create a new device and start it. I click the screen and no response.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @yswtrue,
We are terribly sorry but this public discussion board is restricted to feature support requests. It is not suitable for bug reports or technical support requests.
Please visit our help center at https://support.genymotion.com/hc/en-us/ to get appropriate help.
Thank you for your understanding.
